quippy/quippy/descriptors.py

```diff
@@ -214,6 +214,9 @@ def calc(self, at, grad=False, args_str=None, cutoff=None, **calc_kwargs):
                 descriptor_out[key] = np.concatenate([x.T for x in val])
             elif key == "grad_data":
                 descriptor_out[key] = np.transpose(np.concatenate(val, axis=2), axes=(2, 1, 0))
+            elif key == "ii":
+                # copy cross-index into Numpy arrays (f90wrap exposes pointers)
+                descriptor_out[key] = [np.array(x) for x in val]
```

Comment:
why not `np.copy(val)` to avoid the slow Python loop?

Reply:
or maybe that does need to be `[np.copy(x) for x in val]` - but still good to be explicit that this is a copy, since I'm not sure that `np.array(x)` will always make a copy of a Fortran-hosted array rather than a view/pointer.

Reply:
The tests passed on the first one as well, but let's be safe and indeed use `np.copy()`. The loop is needed, since these arrays may have different sizes depending on the number of neighbours the atoms have.